### DISCUSSION TITLE
Liberar API para acesso externo

### DISCUSSION THREAD

**Discussion by laurolapajunior** [balta-io/forum#597] [Tira Dúvidas] [answered]
uma dúvida, essa API do curso, só funciona internamente para esse projeto FrontEnd, se eu quiser liberar para acesso externo?  para outros FrontEnds, e também se eu quiser adicionar nesta mesma solução um projeto blazor maui, como ficaria configuração da API para atender o APP  também?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @laurolapajunior, como vai?

Não tem restrição na API para consumo externo, qual problema você teve em consumir ela de outros projetos Blazor ou até mesmo MAUI?

Tem algum print para ajudar nisso?

**Comment by laurolapajunior** [comment]
Na verdade achei que tinha, como e setado qual qual URL api e URL front, aí
se eu consumir a api por app por exemplo, api não vai autorizar

Em seg., 24 de jun. de 2024 11:23, Andre Baltieri ***@***.***>
escreveu:

**Reply by andrebaltieri** [reply]
@laurolapajunior, bom dia, como vai?

Não vai autorizar pelo App (Nativo ou Cross-Platform) pois o Cookie é um recurso Web.

Para Apps o melhor seria usar o JWT:
https://balta.io/cursos/seguranca-apis-aspnet-jwt-bearer

**Comment by suarezrafael** [comment]
Se a mesma api fosse projetada para ser usada via app maui e blazor ela já deveria nascer via JWT nesse caso?

**Reply by andrebaltieri** [reply]
Não! Ela pode suportar os dois!
